vdr/vdr.go

```diff
@@ -243,3 +247,25 @@ func (r VDR) resolveControllerWithKey(doc did.Document) (did.Document, crypto.Ke
 
 	return did.Document{}, nil, fmt.Errorf("could not find capabilityInvocation key for updating the DID document: %w", err)
 }
+
+func patchContext(next did.Document) did.Document {
+	NutsContextPresent := false
```

Comment:
Ugly code imo, better to have a function `addContext` which adds it if not present and call it for both contexts?